### ISSUE TITLE
About backdoored_model.py

### ISSUE THREAD

**Issue by zsaba** [ZJZAC/Poison-Ink#8]
we have to train the model with clean train and test data and poisoned data (with train set containing backdoored images (3 percent) in class 0 and all other classes folder and test dataset with all the images with trigger) . This is what i understood from your code. is it correct?
when i make a wm_folder for train_dataset_wm and val_dataset_watermark ,according to the code it is setting the target_wm=0 for all the images in poisoned dataset whose each folder images are set to target class=0.  or we have to take only class 0 which has trigger images from that dataset for train_dataset_wm?

when i trained the backdoored_model.py with original train and testing dataset and poisoned dataset obtained from trigger_generation.py i got best accuracy (68.42) . how can i improve it? what next i have to do?

**Comment by ZJZAC** [comment]
Poisoned data is just the backdoored data, you can pre-define the target label you like(set as 0 by default); for all poisoned data, they share the same target label, while clean data owns their original label.

**Comment by zsaba** [comment]
the dataset we will take for poison is obtained from the trigger_generation.py file?

**Comment by ZJZAC** [comment]
Yes.

**Comment by zsaba** [comment]
thank you for response. one more thing i trained the IN_GE.py with L1 loss and noise  got the trigger and trained the clean and backdoored model then the backdoored model should have ASR=98.48 according to your paper but i got best accuracy=70.1. is it good?
